pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -54,6 +54,10 @@ type PredictorSpec struct {
 	// Model spec for any arbitrary framework.
 	Model *ModelSpec `json:"model,omitempty"`
 
+	// Spec for multiple storage uris.
+	// +listType=atomic
```

Comment:
nit:
```suggestion
	// +listType=atomic
    // +kubebuilder:validation:MinItems=1
```
to make sure users cannot set an empty list